llvm/include/llvm/Analysis/LoopInfo.h

```diff
@@ -392,13 +394,35 @@ class LLVM_EXTERNAL_VISIBILITY Loop : public LoopBase<BasicBlock, Loop> {
     return "<unnamed loop>";
   }
 
+  /// Preserve the induction variable exit value and its debug users by the
+  /// 'indvars' pass if the loop can deleted. Those debug users will be used
+  /// by the 'loop-delete' pass.
+  void preserveDebugInductionVariableInfo(
+      Value *FinalValue, SmallVector<DbgVariableIntrinsic *> DbgUsers) {
```

Comment:
I think it makes sense to delete the DbgUsers parameter and do the `findDbgUsers` call from within this function?

(If you disagree, please update `DbgUsers` to be a const SmallVectorImpl reference or ArrayRef)

Reply:
To avoid adding a dependency on `DebugInfo.h`, making `DbgUsers` to be `const SmallVectorImpl reference`.